exonum-java-binding-core/rust/src/utils/mod.rs

```diff
@@ -20,6 +20,7 @@ mod errors;
 mod exception;
 mod handle;
 mod jni;
+pub mod jni_cache;
```

Comment:
I declared the module this way because the intention was to use cached entities in other modules as:
```
use jni_cache;
...
jni_cache::get_some_cached_entity()
```
@vitvakatu WDYT - is it a good approach?